### PR TITLE
docs: include UI Kit version 0.10.0

### PR DESCRIPTION
**Pull request by droshev**

### CODE REVIEW

**Review by InnaAtanasova — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -185,7 +185,7 @@ <h2 class="section__title">Design</h2>
           Sketch UI Kit bundle includes ready-to-use, state-of-the-art drafts of SAP layouts, patterns and components in Sketch. You are welcome to use the UI kit to visualize your SAP app. They are easy to use and give you a realistic impression of your final design.
```
Comment (jessmetivier) [inline docs/index.html:185]: **Quick edit**
Our UI Kit bundle includes ready-to-use, state-of-the-art drafts of SAP layouts, patterns and components in Sketch. You are welcome to use the UI kit to visualize your SAP app. They are easy to use and will provide a realistic impression of your final designs.

File: docs/index.html [hunk 2]
```diff
@@ -185,7 +185,7 @@ <h2 class="section__title">Design</h2>
...
         </p>
         <div class="section__container-button">
-          <a role="button" class="section__button" href="./images/ui-kit/Fundamental_Library_UI_Kit_V.0.9.0.zip">Download</a>
+          <a role="button" class="section__button" href="./images/ui-kit/Fundamental_Library_UI_Kit_V.0.10.0.zip">Download</a>
         </div>
         <h3 class="section__subtitle">What's in UI Kit bundle?</h3>
```
Comment (jessmetivier) [inline docs/index.html:190]: **Quick edit**
What's in the UI Kit bundle?

File: docs/index.html [hunk 3]
```diff
@@ -196,7 +196,7 @@ <h3 class="section__subtitle">What's in UI Kit bundle?</h3>
                 alt="Sketch Logo"
                 class="section__logo"/>
 
-            <h3 class="section__item-title">Sketch UI Kit v0.9.0 </h3>
+            <h3 class="section__item-title">Sketch UI Kit v0.10.0 </h3>
             <p class="section__text">Our Sketch file has over 30 unique components refined and streamlined to allow any user the ability quickly and effeciently build Fiori 3 interfaces using the Fundamental Library.</p>
```
Comment (jessmetivier) [inline docs/index.html:200]: **Quick edit**
Our Sketch file has over 30 unique components that are refined and streamlined to empower anyone using Fundamental Library with the ability to quickly and efficiently build Fiori 3 interfaces.

**Review by jessmetivier — COMMENTED** [review]
I think these changes to the text are important, other edits/designs will be discussed tomorrow/Friday.

File: docs/index.html (hunk 2 above)
Comment (droshev) [reply docs/index.html:190]: font 72, icons, and ui kit

**Pull request merged**